09-intermediate-rails/10-creating-apis.md

```diff
@@ -1,33 +1,47 @@
 # Creating an API
 
 ## Learning Goals
-- Apply our Rails knowledge to a new construct
-- See how to render results other than views
-- Use status codes to communicate outcomes
+Students should be able to...
+
+- Compare and contrast web applications and APIs
+- Develop a Rails API to provide CRUD functionality
+- Use HTTP status codes to communicate outcomes
 
 ### Introduction
 Last week, we learned about using APIs that already exist. Now, our goal is to create APIs ourselves.
 
-When creating an API, usually the goal is to provide data to a consumer, whether that be a human (not normally) or another system (usually). Many of the applications that we have created previously could have been APIs instead of web applications.
+When creating an API, usually the goal is to provide data to a consumer, whether that be a human (not normally) or another system (usually). This could be a mobile app, a single-page application written in JavaScript, or something else entirely.
+
+#### Vocabulary
+| Word | Definition | Example
+|------|------------|--------
+| Resource | Some piece of data we want to share with the world | A single pet, the list of all pets
+| Endpoint | A unique URL + HTTP verb that provides access to some resource | `GET /pets/7`
+| Client   | Who- or whatever is consuming our API | A mobile app
 
-### Getting Started
+### APIs vs Web Apps
 When we think about using an API vs using a web application, what is the key difference? **HTML** for humans to consume versus **JSON** for systems to consume.
```

Comment:
I know you didn't modify this - but I think we want some more _is_s in this last sentence to add some clarity